### PR TITLE
appids: Show the original identifier (per the user) as well as the normalized one

### PR DESCRIPTION
**Pull request by Jongy**
## Description
For better user experience - show the "identifier" as the user has written it, in addition to showing the normalized one.

TODOs:
* [x] uwsgi appid tests
* [x] uwsgi `--ini-paste` support

### CODE REVIEW

File: gprofiler/metadata/application_identifiers.py [hunk 1]
```diff
@@ -73,10 +73,9 @@ def _append_python_module_to_proc_wd(process: Process, module: str) -> str:
 
 
 def _append_file_to_proc_wd(process: Process, file_path: str) -> str:
-    if os.path.isabs(file_path):
-        return os.path.realpath(file_path)
-
-    return os.path.realpath(os.path.join(process.cwd(), file_path))
+    # if file_path is absolute, then the process.cwd() is removed.
+    file_path = os.path.join(process.cwd(), file_path)
+    return resolve_proc_root_links(f"/proc/{process.pid}/root", file_path)
```
Comment (guybortnikov) [inline gprofiler/metadata/application_identifiers.py:78]: Strip proc_root

**Review by guybortnikov — CHANGES_REQUESTED** [review]
.

**Comment by Jongy** [comment]
@guybortnikov GH is retarded and doesn't display the diff of the tests file properly, use this instead: `git diff master:tests/test_appid.py origin/appid-improvements:tests/test_appids.py` (after `gill pull`)

File: gprofiler/metadata/application_identifiers.py [hunk 2]
```diff
@@ -133,49 +134,53 @@ def get_application_name(self, process: Process) -> Optional[str]:
...
+        config_file = _get_cli_arg_by_name(cmdline, "--ini", check_for_equals_arg=True)
+        if config_file is _NON_AVAILABLE_ARG:
+            config_file = _get_cli_arg_by_name(cmdline, "--ini-paste", check_for_equals_arg=True)
+
+        if config_file is _NON_AVAILABLE_ARG:
+            return None, None
 
         config = configparser.ConfigParser(strict=False)
-        config.read(resolve_host_path(process, config_file))
+        config.read_file(cls._open_uwsgi_config_file(process, config_file))
```
Comment (guybortnikov) [inline gprofiler/metadata/application_identifiers.py:154]: Close the file in `finally` block (or either use some contextmanager on your `_open_uwsgi_config_file`) 
I know you're gonna tell me GC will clean it but it's a bad practice.

**Review by guybortnikov — CHANGES_REQUESTED** [review]
Looks good, small comment

**Review by guybortnikov — APPROVED** [review]

**Pull request merged**